debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/SqlUtils.java

```diff
@@ -129,6 +328,7 @@ private static String buildTableInPredicate(List<String> tables) {
 
         StringJoiner tableNames = new StringJoiner(",");
         tables.forEach(table -> tableNames.add("'" + table + "'"));
+        // todo: discuss with Andrey about partition tables here
```

Comment:
@AndreyIg This was specifically shown by an upstream contributor to be problematic for partition tables, which we've since changed.  I wanted to highlight this in case you wanted to pull this in in your fork for now.

Reply:
Noted, good point. our customers have no partitioned tables yet